src/components/Select/types.ts

```diff
@@ -148,4 +157,13 @@ export type SelectClearProps = SelectClearIconProps & {
     onMouseLeave: (e: React.MouseEvent) => void;
 };
 
+export type SelectCounterProps = {
+    /** number of selected elements to show */
+    count: number;
```

Comment:
We use and manage this value only inside our code, so it could be only number in this usage.

https://github.com/gravity-ui/uikit/pull/1368/files#diff-9cb10225f1e530d44628bfef03a4ff5077dbdc94f3a7f7a10ce6fbeeec76b64dR118

Reply:
I suppose we use in here https://github.com/gravity-ui/uikit/pull/1368/files#diff-c1510619d5ba5e80032efb9ba67b29e63307d6a80b81fc2cd3edc466a05d0439R57 as a public interface?

Reply:
Yes we pass this parameter to render props. Ok, let's make it string if it is more convenient)
https://github.com/gravity-ui/uikit/pull/1368/commits/fb78910e310747666144958cefc8542fc18ca8cd

Reply:
No, I suggest to use type `string | number` to have opportunity to set values `1`, `1k` an so on

Reply:
Ok, remade it) https://github.com/gravity-ui/uikit/pull/1368/commits/d7ed58c8a45a59afe4621a052ddc66010e45dfda

Reply:
return number) https://github.com/gravity-ui/uikit/pull/1368/commits/00f69d63d16ef783e99ffbff826b724e558dbb5d